### PR TITLE
My ref  topic

### PR DESCRIPTION
**Pull request by writingteacher**

### CODE REVIEW

File: api/to-do-db-source.json
Comment (rbwatson) [inline api/to-do-db-source.json:1]: Please remove this file from the pull request

File: docs/api/user.md [hunk 1]
```diff
@@ -9,9 +9,7 @@ Base endpoint:
 {server_url}/users
 ```
 
-Contains information about the users of the service.
-
-To have a task in the service, the user must be added to
+Contains information about the users of the service. To have a task in the service, the user must be added to
```
Comment (rbwatson) [inline docs/api/user.md]: I try to start new sentences on a new line to make future revisions easier to spot. Changes to the 2nd, 3rd, 4th... sentence are easier to spot and compare when the lines don't wrap in the display.

File: docs/index.md [hunk 2]
```diff
@@ -42,3 +42,6 @@ generally `http://localhost:3000`.
 
 * [user resource](api/user)
 * [task resource](api/task)
+
+ABC reference. Ready to go.
```
Comment (rbwatson) [inline docs/index.md]: I'm not sure what this refers to. Can you explain?

File: docs/api/users-change-user-email.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+---
+
+# Update user email
+
+The Patch request updates the user's email address but does not change the other parameters in the [`user`](user) object.
```
Comment (rbwatson) [inline docs/api/users-change-user-email.md]: With this reference topic, would there also be others like, Update user last_name, Update user first_name?
Or, could you change the topic title to **Update user properties** and show how what you describe in this topic for the email address could apply to any of the parameters by including the parameters that you want to change in the request body?

File: docs/api/users-change-user-email.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+## Method
+PATCH
+
+
+## Params
+
+| Parameter name | Type | Description |
+| -------------- | ------ | ------------ |
+| `email` | string | The user's new email address that will replace the address in the current record. |
+| `id` | number | The user's unique record ID. |
```
Comment (rbwatson) [inline docs/api/users-change-user-email.md]: Is this required if you have it in the URL?

File: docs/api/users-change-user-email.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+## Request headers
+
+None
+
+## Request body
+
+{
+	"email":"marty2@example.com"
+}
```
Comment (rbwatson) [inline docs/api/users-change-user-email.md]: This example is different from what is listed in **Params** above. Is that intentional?
If `id` is is optional, then that should be mentioned in the **Params** section.

File: docs/api/users-change-user-email.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+## Request example
+
+cURL 
+```
+curl --location --request PATCH 'localhost:3000/users/3' \
+--header 'Content-Type: application/json' \
+--data-raw '    {
+        "email": "marty2@example.com"
+    }'
+```
```
Comment (rbwatson) [inline docs/api/users-change-user-email.md]: Great example!

**Review by rbwatson — CHANGES_REQUESTED** [review]
This is pretty good!
Please review my comments here and in https://canvas.uw.edu/courses/1715843/discussion_topics/8949983 and update the PR.
Thanks!

**Comment by rbwatson** [comment]
Did you want to close the pull request or just indicate that you resolved the changes?
If the latter, please re-open the PR so I can merge it.
Thanks!

**Comment by writingteacher** [comment]
Thanks Bob. I will close the pull request and try again.

**Pull request closed without merging**